src/test/java/project/EvenOddNumbersCheckerTest.java

```diff
@@ -0,0 +1,24 @@
+package project;
+
+import org.junit.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class EvenOddNumbersCheckerTest {
+
+    @Test
+    public void shouldTestEvenNumbers() {
+        EvenOddNumbersChecker checker = new EvenOddNumbersChecker();
+        assertTrue(checker.isEven(2));
```

Comment:
Ponownie sprawdzasz kilka razy tę samą rzecz. Jeśli uważasz, że dodatkowe sprawdzanie jest z jakiegoś powodu potrzebne, to napisz do tego osobny test. Jednak w tym przypadku nie jest to potrzebne. Wystarczy pozbyć się dwóch sprawdzeń. Już jedno jest wystarczające, aby być pewnym, że wszystko działa poprawnie :)